SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbarCommands.cpp

```diff
@@ -10,9 +10,9 @@ void FSpatialGDKEditorToolbarCommands::RegisterCommands()
 	UI_COMMAND(CreateSpatialGDKSchemaFull, "Schema (Full Scan)", "Creates SpatialOS Unreal GDK schema for all assets.", EUserInterfaceActionType::Button, FInputGesture());
 	UI_COMMAND(DeleteSchemaDatabase, "Delete schema database", "Deletes the scheme database file", EUserInterfaceActionType::Button, FInputGesture());
 	UI_COMMAND(CreateSpatialGDKSnapshot, "Snapshot", "Creates SpatialOS Unreal GDK snapshot.", EUserInterfaceActionType::Button, FInputGesture());
-	UI_COMMAND(StartNoAutomaticConnection, "Start", "Not Automatic Connect to service", EUserInterfaceActionType::Button, FInputGesture());
-	UI_COMMAND(StartLocalSpatialDeployment, "Local", "Connect to local deployment service", EUserInterfaceActionType::Button, FInputGesture());
-	UI_COMMAND(StartCloudSpatialDeployment, "Cloud", "Connect to cloud SpatialOS service", EUserInterfaceActionType::Button, FInputGesture());
+	UI_COMMAND(StartNoAutomaticConnection, "None", "Doesn't automatically connect to SpatialOS", EUserInterfaceActionType::Button, FInputGesture());
+	UI_COMMAND(StartLocalSpatialDeployment, "Local", "Autoamtically connects to a local deployment", EUserInterfaceActionType::Button, FInputGesture());
+	UI_COMMAND(StartCloudSpatialDeployment, "Cloud", "Automatically connects to a cloud deployment", EUserInterfaceActionType::Button, FInputGesture());
```

Comment:
```suggestion
	UI_COMMAND(StartCloudSpatialDeployment, "Cloud", "Start a cloud deployment", EUserInterfaceActionType::Button, FInputGesture());
```

Reply:
Not sure I want to change it to that because on mac it is definitely only the connecting part. no starting of a cloud deployment

Reply:
Do you know the thing about how the human brain can read the word if the first and last letter is correct but all the rest are scrambled?

Reply:
As mentioned in the standup, this will be the tooltip for the buttons that start the local and cloud deployments. On Mac, this button will be disabled.

Reply:
Do you konw the tihng auobt how the hmaun bairn can raed the wrod if the frsit and lsat lteetr is crcoret but all the rset are smralebcd?

:-)